src/problems_truss.jl

```diff
@@ -0,0 +1,65 @@
+# This file is a part of JuliaFEM.
+# License is MIT: see https://github.com/JuliaFEM/JuliaFEM.jl/blob/master/LICENSE.md
+
+# This is copied from the files problem_elasticity.jl
+"""
+This truss fromulation is from
+"""
+type Truss <: FieldProblem
+    # these are found from problem.properties for type Problem{Elasticity}
+    formulation :: Symbol
```

Comment:
Actually I think we don't need even this at this point, type definition can be empty:
```
type Truss <: FieldProblem
end
```

Reply:
Later on we can define here for example, how do we want to calculate our local stiffness matrix. For example, we can have `finite_strain :: Bool` and access this inside assembly procedure from `problem.properties.finite_strain` and in that case calculate also geometric stiffness and have finite strain truss element.